### PR TITLE
W001w

### PR DESCRIPTION
**Pull request by Holado**
Option added for skipping spelling suggestion
MAL-59

### CODE REVIEW

File: src/reynir_correct/errtokenizer.py [hunk 1]
```diff
@@ -1576,6 +1576,7 @@ def lookup_unknown_words(
     token_stream: Iterable[CorrectToken],
     only_ci: bool,
     apply_suggestions: bool,
+    sss: bool,
```
Comment (vthorsteinsson) [inline src/reynir_correct/errtokenizer.py:1579]: We normallu use pretty descriptive parameter names, so I suggest renaming this to something more meaningful, even if it is long.

File: src/reynir_correct/main.py [hunk 2]
```diff
@@ -141,6 +141,12 @@ class AnnResultDict(TypedDict):
     action="store_true",
 )
 
+parser.add_argument(
+    "skip_spelling_suggestion",
+    "--sss",
+    action="store_true",
+    help="Create an analysis report for token results",
```
Comment (vthorsteinsson) [inline src/reynir_correct/main.py:148]: Not sure I understand this help text. Can it be made clearer?

File: src/reynir_correct/errtokenizer.py (hunk 1 above)
Comment (Holado) [reply src/reynir_correct/errtokenizer.py]: Done!

File: src/reynir_correct/main.py (hunk 2 above)
Comment (Holado) [reply src/reynir_correct/main.py]: Done!

**Pull request merged**